test/e2e/gui/elements/object.py

```diff
@@ -100,7 +100,7 @@ def click(
             x: int = None,
             y: int = None,
             button=None,
-            timeout=1
```

Comment:
we set default for 1 in general. in the places where it waits more than one, we go there and put the timeout explicitly there , can we change that?

Reply:
with timeout=5 we can filter the slowest parts of the app, address them first and then set it back to 1 once major freezes are fixed
otherwise we will end up with too many custom timeouts per element / action

Reply:
IMO each custom timeout should have a corresponding GitHub task to fix the perf. This is the true value of the timeout 😄 If we have this then maybe it's not bad to have too many custom timeouts per action.

Reply:
Ohh, I've just realised this is the default timeout! Please keep it as low as possible! Based on our current goals we'll need to reach sub second default in the next months!